pkg/webservice/routes.go

```diff
@@ -127,6 +127,13 @@ var webRoutes = routes{
 		getContainerHistory,
 	},
 
+	route{
+		"Scheduler",
+		"GET",
+		"/ws/v1/partition/{partition}/queues",
+		getPartitionQueues,
```

Comment:
Now we have two endpoint for retrieving the queue infos. Shouldn't we delete the old one? Or what is the plan, how will the transition to the new API will be done?

Reply:
Idea is to migrate the users to newly designed API's and slowly retire old one's. Hence, didn't touched the old places, in fact created a completely new dao's, struct etc and planning to take care of all clean up activities after the adoption through separate jira's. This way, we need not worry to about backward compatibility etc. Hope it makes sense.